ui-v2/app/templates/dc/nodes/show.hbs

```diff
@@ -42,14 +47,14 @@
         {{#each
             (compact
                 (array
-                    (hash id=(slugify 'Health Checks') partial='dc/nodes/healthchecks')
-                    (hash id=(slugify 'Services') partial='dc/nodes/services')
-     (if tomography (hash id=(slugify 'Round Trip Time') partial='dc/nodes/rtt') '')
-                    (hash id=(slugify 'Lock Sessions') partial='dc/nodes/sessions')
+     (if (gt item.Checks.length 0)  (hash id=(slugify 'Health Checks') partial='dc/nodes/healthchecks') '')
+                                    (hash id=(slugify 'Services') partial='dc/nodes/services')
+     (if tomography                 (hash id=(slugify 'Round Trip Time') partial='dc/nodes/rtt') '')
+                                    (hash id=(slugify 'Lock Sessions') partial='dc/nodes/sessions')
```

Comment:
I thought the Health Check tab was disappearing if there were no health checks, but I think we are going to show a 'no health checks' message instead - so there'll be another commit or 2 on top of this yet 😂 . If you get here before I've done that, you can probably ignore a lot of this.

Reply:
The commits on top of this are done now so the above has gone, please ignore!